reana_workflow_engine_serial/tasks.py

```diff
@@ -162,7 +162,7 @@ def run(
             publisher,
             workflow_uuid,
         )
-        if status != "succeeded":
+        if status != "finished":
```

Comment:
Not related to the task at all, just speaking my mind. 

Maybe we should move `JobStatus` and `WorkflowStatus` enums to r-commons so we can use them here directly? If we had had this logic, we would have avoided many PRs for this task.

Reply:
Definitely, I agree it is not practical to have the enums in REANA-DB, we have the ticket https://github.com/reanahub/reana-workflow-controller/issues/204 describing it. It is true I thought of it while implementing but the number of changes would have been bigger with higher chances of introducing a bug just before the workshop.


We should give it more priority for next time since it will make the work on future workflow engines easier too.